staging/src/k8s.io/client-go/tools/clientcmd/api/v1/types.go

```diff
@@ -209,6 +209,11 @@ type ExecConfig struct {
 	// Preferred input version of the ExecInfo. The returned ExecCredentials MUST use
 	// the same encoding version as the input.
 	APIVersion string `json:"apiVersion,omitempty"`
+
+	// InstallHint is a blurb of text to indicate how to acquiare the binary
+	// (i.e., from Command) for this exec provider.
+	// +optional
+	InstallHint string `json:"installHint,omitempty"`
```

Comment:
Make this match the KEP (either by updating the KEP or here).

Reply:
Done.